gossipd/routing.c

```diff
@@ -1208,6 +1210,7 @@ bool routing_add_channel_update(struct routing_state *rstate,
 	struct unupdated_channel *uc;
 	u8 direction;
 	struct amount_sat sat;
+	bool spam;
```

Comment:
```suggestion
	bool spam = false;
```

In this way you avoid to several else branch only for the spam, if the code keep growing, the state can be difficult to track

Reply:
I was trying to be cognizant of the style guide, but it probably makes more sense to do it this way for the reader's sake.